### DISCUSSION TITLE
Some icons aren't downloadable

### DISCUSSION THREAD

**Discussion by ghost** [elrumo/macOS_Big_Sur_icons_replacements#1226] [Help with the website] [answered]
Like the title says. Some icons appear to not be downloadable. They only appear on the main page with all the recent icons but don't show up in search results or the uploader's page. For example the icon for Amphetamine Enhancer.

**Comment by elrumo** [comment, marked as answer]
This one? It worked for me, what happens if you right click and copy the link address?
![image](https://user-images.githubusercontent.com/36299870/123788416-e4027f00-d8d3-11eb-9078-e37f74364457.png)

Copy link address ->
![image](https://user-images.githubusercontent.com/36299870/123788477-f8df1280-d8d3-11eb-883d-8a7c45a132e5.png)

**Comment by ghost** [comment]
Not that one. This one. If I right click this one it saves as a png file
(if I remember correctly).


<img width="598" alt="Screen Shot 2564-06-30 at 13 27 10" src="https://user-images.githubusercontent.com/85490680/123912235-fa2d3f80-d9a6-11eb-899e-7ff2036252bc.png">

**Reply by elrumo** [reply]
I can't find that particular icon on the site, can you? If not, it's most likely because it was lost after the hack and all we have available now is the low res image.

**Reply by ghost** [reply]
It showed up a week ago as a recently uploaded icon but wasn't downloadable and would not appear in subsequent searches for the icon. So I guess you're right, thanks Elrumo.
